ape_safe/accounts.py

```diff
@@ -13,18 +13,22 @@
 from ape.utils import ZERO_ADDRESS, cached_property
 from ape_ethereum.transactions import TransactionType
 from eip712.common import create_safe_tx_def
+from eip712.messages import hash_eip712_message
 from eth_utils import keccak, to_bytes, to_int
 from ethpm_types import ContractType
 
-from ape_safe.client import SafeClient, SafeTx
+from ape_safe.client import BaseSafeClient, MockSafeClient, SafeClient, SafeTx, SafeTxConfirmation
 from ape_safe.exceptions import (
     ClientUnavailable,
     NoLocalSigners,
     NotASigner,
     NotEnoughSignatures,
+    SafeClientException,
     handle_safe_logic_error,
 )
 
+from .utils import order_by_signer
```

Comment:
```suggestion
from ape_safe.utils import order_by_signer
```

because we already have those types of imports above and this needlessly creates a new section